MyHashMap.java

```diff
@@ -0,0 +1,100 @@
+// Time Complexity : o(1)
```

Comment:
Please check the time complexity, if you are searching through the linked list then how can it be O(1)!?